co-simulation/tools/geneate_background_traffic.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#This script will generate random traffic to be used in Sumo-carla co simulation for a any Sumo Network File given the simulation duration, number of vehicles and a random number seed
+# This script requires  installation of Sumo tools randomTrips.py and Python
+echo "Sumo Network Files in this directoryi:"
+ls *.net.xml
+# Prompt for user inputs
+echo 
+echo "Enter the Sumo Network File Name:"
+read net_file
+
+echo "Output File Name:"
+read out_file
+
+echo "Enter Duration of the Simulation:"
+read sim_duration
+
+echo "How many vehicles you want to instantiate?:"
+read veh_number
+
+echo "Enter Random Number Seed (Different Seed will genrate different traffic):"
+read seed
+
+# Calculate the rate of vehicle generation
+rate=$(echo "$sim_duration / $veh_number" | bc -l)
```

Comment:
Minor thought, but might be worth making these command-line arguments, and then taking interactive can be a fallback if not all the arguments are passed in. Doesn't have to be fancy even, could just assume that the args are always passed in the same order. And then if the count of args doesn't match expected, you run this interactive logic.

Only mentioned because this might conceivably be integrated eventually as part of a larger tool or automated pipeline and passing commandline arguments works way better for that than interactive input does.